.github/workflows/bc.yml

```diff
@@ -3,6 +3,16 @@ name: bc
 on:
   workflow_call:
     inputs:
+      composer-root-version:
+        description: Composer root version.
+        default: dev-master
+        required: false
+        type: string
+      subpackage:
```

Comment:
The name is a bit confusing. `db` is the base package and AR is sub-package, but this key is used in AR so it reads as "db is sub-package of AR".

Reply:
Actually it is a *required* package which will be installed using `composer require` command. Rename to `required-package`?

Reply:
Yes. That is way cleaner.

Reply:
Done